_docs/quickstarts/forseti-security/index.md

```diff
@@ -4,30 +4,58 @@ order: 002
 ---
 # {{ page.title }}
 
-This quickstart explains how to use the Forseti setup wizard to get started
-quickly with Forseti Security for Google Cloud Platform (GCP) projects.
+This quickstart explains how to use the Forseti setup wizard, which helps to 
+automate some of the Forseti installation and setup process.
+
+Setup wizard is not available prior to version 1.1.0.
 
 ## Before you begin
 
-To complete this quickstart, you will need:
+Prior to running the setup wizard, you will need:
 
   - A GCP organization for which you want to deploy Forseti.
   - Org Admin IAM role in order for the script to assign the Forseti 
   service account roles on the organization IAM policy.
+  - A GCP project dedicated to Forseti.
+  - Enable billing on the project.
+
```

Comment:
Add a bullet:
`- The latest version of [Cloud SDK](https://cloud.google.com/sdk/downloads)`